docs/htdocs/info/docs/api/variation/variation_tutorial.html

```diff
@@ -919,18 +934,20 @@ <h2 id="variant_sets">Variant sets</h2>
 </div>
 
 
-<br /><hr />
 <h2 id="ag" style="font-size:large">Alleles and Genotypes</h2>
 
 <div style="width:800px" class="info">
   <h3>Requirement for 1000 Genomes data and other large genotyping projects</h3>
   <div class="message-pad">
-    <p>1000 Genomes Project phase 3 data is accessed via VCF files stored on Ensembl's FTP server. The API makes this access seamless, though it requires you install some extra software dependencies detailed on the <a href="/info/docs/api/api_installation.html#variation">API installation page</a>. See also this <a href="http://www.ensembl.info/blog/2015/06/18/1000-genomes-phase-3-frequencies-genotypes-and-ld-data/" rel="external">blog post</a>.
+    <p>1000 Genomes Project phase 3 data is accessed via VCF files stored on Ensembl's FTP server. 
+    The API makes this access seamless, though it requires you install some extra software dependencies detailed on the 
+    <a href="/info/docs/api/api_installation.html#variation">API installation page</a>. 
+    See also this <a href="http://www.ensembl.info/blog/2015/06/18/1000-genomes-phase-3-frequencies-genotypes-and-ld-data/" rel="external">blog post</a>.
     </p>
   </div>
 </div>
 
-<h2 id="alleles">Alleles and frequencies</h2>
+<h3>Alleles and frequencies</h3>
```

Comment:
Most variants in the Ensembl Variation database have associated allele frequencies. => Many variants available through the Ensembl Variation API have allele frequency information.